source/extensions/extensions_build_config.bzl

```diff
@@ -69,6 +69,7 @@ EXTENSIONS = {
     "envoy.filters.network.ratelimit":                  "//source/extensions/filters/network/ratelimit:config",
     "envoy.filters.network.tcp_proxy":                  "//source/extensions/filters/network/tcp_proxy:config",
     "envoy.filters.network.thrift_proxy":               "//source/extensions/filters/network/thrift_proxy:config",
+    "envoy.filters.network.rbac":                       "//source/extensions/filters/network/rbac:config",
```

Comment:
nit: please alpha order (can you check to see if anything else is not ordered)

Reply:
Also reordered HTTP filters.